css/includes/components/_kanban.scss

```
@@ -181,16 +181,22 @@
                         .kanban-column-header-content {
                             flex-direction: column;
 
-                            .kanban-collapse-column,
-                            .kanban-column-title,
-                            .kanban_nb {
+                            .kanban-collapse-column {
                                 transform: rotate(90deg);
                                 transform-origin: center;
                                 display: inline-block;
                                 margin: calc(50% - 8px) 0;
                                 white-space: nowrap;
                             }
 
+                            .kanban-column-title,
+                            .kanban_nb {
+                                writing-mode: tb;
```

According to https://developer.mozilla.org/en-US/docs/Web/CSS/writing-mode#tb, the `tb` value is deprecated.
```suggestion
                                writing-mode: vertical-lr;
```